support-frontend/assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -97,6 +108,82 @@ const iconForCountryGroup = (countryGroupId: CountryGroupId): React$Element<*> =
   }
 };
 
+const localisedAverageAmountSentence: {
+  [CountryGroupId]: {
+    amountSentence: string,
+    averageAnnualAmount: number,
+  }
+} = {
+  GBPCountries: {
+    amountSentence: 'In the UK, the',
+    averageAnnualAmount: 50.37,
+  },
+  UnitedStates: {
+    amountSentence: 'In the US, the',
+    averageAnnualAmount: 50.16,
+  },
+  AUDCountries: {
+    amountSentence: 'In Australia, the',
+    averageAnnualAmount: 82.65,
+  },
+  EURCountries: {
+    amountSentence: 'In Europe, the',
+    averageAnnualAmount: 52.92,
+  },
+  NZDCountries: {
+    amountSentence: 'In New Zealand, the',
+    averageAnnualAmount: 58.51,
+  },
+  Canada: {
+    amountSentence: 'In Canada, the',
+    averageAnnualAmount: 62.68,
+  },
+  International: {
+    amountSentence: 'The',
+    averageAnnualAmount:  61.78,
+  },
+};
+
+
+function averageAmountVariantCopy(
+  countryGroupId: CountryGroupId,
+  contributionType: ContributionType,
+  currencyString: string,
+) {
+  const localAverageAmountInfo = localisedAverageAmountSentence[countryGroupId];
+  const localAverageAmount = Math.round(localAverageAmountInfo.averageAnnualAmount);
```

Comment:
I'm not keen on rounding it especially as it ends up as exactly 50 in the UK and US